Plugins/DataSource/Barometer/Temperature.cs

```diff
@@ -14,18 +14,16 @@ public class Temperature : Barometer
         /// Read value from sensor and prepare real value
         /// </summary>
         /// <returns>Air pressure or 0.0</returns>
-        protected override MeasurementValue ValueGetRaw()
+        protected override MeasurementValue RawValue()
         {
             MeasurementValue result = new MeasurementValue(Name, Unit, TemperatureConfig.Instance.SortOrder);
 
             if (_Bricklet == null)

```

Comment:
This should check `_Bricklet != null` - otherwise the temperature will never be read from the sensor.

Reply:
thanks for catching that!